### PR TITLE
Cleaning up the docs

### PR DESCRIPTION
**Pull request by ampolloreno**

### CODE REVIEW

File: docs/qaoa/qaoa_overview.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Structure
 _________
 
 The pyQAOA package contains separate modules for each type of problem
-instance: MAX-CUT, K-SAT, etc.
+instance: MAX-CUT, graph partitioning, etc.
 Each problem instance is a class that inherits the main QAOA object and
```
Comment (ncrubin) [inline docs/qaoa/qaoa_overview.rst]: This is no longer how the code is structured.  No override is necessary. 

maybe something like...."For each problem instance the user specifics the driver Hamiltonian, cost Hamiltonian, and the approximation order of the algorithm."

**Comment by ncrubin** [comment]
Small changes and then ready to merge.

**Comment by ampolloreno** [comment]
@ncrubin I don't see them, I've added you as a reviewer.

**Comment by ampolloreno** [comment]
@ncrubin Changes are done.

**Comment by ncrubin** [comment]
Merged #12

**Pull request merged**